src/components/GisidaLite/helpers.tsx

```diff
@@ -90,3 +94,34 @@ export const gsLiteStyle: Style | string = DIGITAL_GLOBE_CONNECT_ID
       version: 8,
     }
   : 'mapbox://styles/mapbox/satellite-v9';
+
+/**
+ * gets map zoom, center and bounds form structures or jurisdictions
+ * @param {StructureFeatureCollection | null} structures - structure points
+ * @param {Jurisdiction | null} jurisdiction - jurisdiction
+ */
+export const getZoomCenterAndBounds = (
+  structures: StructureFeatureCollection | null,
+  jurisdiction: Jurisdiction | null
+) => {
+  let mapCenter;
+  let mapBounds;
+  let zoom;
+  if (structures?.features?.length) {
+    mapBounds = GeojsonExtent(structures);
+  }
+  if (!mapBounds && jurisdiction?.geojson) {
+    mapBounds = GeojsonExtent(jurisdiction.geojson);
+  }
```

Comment:
This means that if both the jurisdiction and the structures are defined then we will use the jurisdiction to get the center and zoom.  Could we make it the other way round (which is how it is currently).  i.e. that the structures will override the jurisdiction if both exist

Reply:
we will only use jurisdiction if structures are not defined. That is why on line 113 am first checking if `mapBounds`  is defined.